client/my-sites/checkout/checkout-thank-you/licensing-thank-you-auto-activation.tsx

```diff
@@ -148,102 +143,87 @@ const LicensingThankYouAutoActivation: FC< Props > = ( {
 	const selectedItem = selectDropdownItems.find( ( item ) => item.props.selected );
 
 	return (
-		<Main fullWidthLayout className="licensing-thank-you-auto-activation">
+		<>
 			<PageViewTracker
 				options={ { useJetpackGoogleAnalytics: true } }
 				path={ '/checkout/jetpack/thank-you/licensing/:product' }
```

Comment:
Should `path` be `/checkout/jetpack/thank-you/licensing-auto-activate/:product` instead?